src/track/playcounter.cpp

```diff
@@ -0,0 +1,22 @@
+#include "track/playcounter.h"
+
+
+void PlayCounter::updatePlayed(bool bPlayed) {
+    if (bPlayed) {
+        // increment the play count when play during the current session
```

Comment:
Ups, no. The played flag is indeed stored in the database! The handling of 
the play count stuff is really strange.

On 11/14/2015 01:20 PM, Uwe Klotz wrote:
> I've tried to preserve the behaviour of the flag and the counter as I 
> found it in the existing code. But now I've noticed that this 
> implementation was always wrong!
>
> We need to distinguish between the flag that is only relevant during the 
> current session and the count that is stored persistently in the library. 
> I will update the code, the comments, and do some renamings as you suggested.
>
> On 11/12/2015 09:11 PM, Daniel Schürmann wrote:
>>
>> In src/track/playcounter.cpp 
>> <https://github.com/mixxxdj/mixxx/pull/772#discussion_r44707976>:
>>
>> > @@ -0,0 +1,22 @@
>> > +#include "track/playcounter.h"
>> > +
>> > +
>> > +void PlayCounter::updatePlayed(bool bPlayed) {
>> > +    if (bPlayed) {
>> > +        // increment the play count when play during the current session
>>
>> "when" is confusing here, because the code is not conditional.
>> Does the comment add any value?
>> We may consider to rename m_iTimesPlayed instead.
>> How about "m_timesPlayedCurrentSession"
>>
>> —
>> Reply to this email directly or view it on GitHub 
>> <https://github.com/mixxxdj/mixxx/pull/772/files#r44707976>.
>>
>